api/handler/delete.go

```diff
@@ -225,7 +228,7 @@ func (h *handler) DeleteMultipleObjectsHandler(w http.ResponseWriter, r *http.Re
 				deletedObj.DeleteMarker = true
 			}
 			response.DeletedObjects = append(response.DeletedObjects, deletedObj)
-		} else if !requested.Quiet {
+		} else {
```

Comment:
I suppose  have  the following condition:
```
if obj.Error != nil {
    // fill `response.Errors`
} else if !requested.Quite {
    // fill `response.DeletedObjects`
}
```

Reply:
But we need to consider case when `len(obj.VersionID) == 0 && errors.IsS3Error(obj.Error, errors.ErrInvalidVersion)` as not an error in both modes `quite` and `verbose`. That's why I think, first of all, we should go to branch `obj.Error == nil`.

Reply:
https://github.com/nspcc-dev/neofs-s3-gw/pull/392#discussion_r841702616